single_level.py

```diff
@@ -90,22 +90,64 @@ def makeRow(resource):
 		else:
 			row.append("false")
 	
-	print row
+	print row	
 
 def main():
-	print "Creating a csv"
+	#Get list of resources
+	resourceIds = requests.get(repositoryBaseURL + "/resources?all_ids=true", headers=headers)
+	
+	#Creating csv
 	writer = csv.writer(open(spreadsheet, "wb"))
-	column_headings = ["title", "resource", "extent", "date", "language", "repository", "creator", "scope", "restrictions"]
+	column_headings = ["title", "publish", "resource", "extent", "date", "language", "repository", "creator", "scope", "restrictions"]
 	writer.writerow(column_headings)
-
-	print "Getting a list of resources"
-	resourceIds = requests.get(repositoryBaseURL + "/resources?all_ids=true", headers=headers)
-	print "Writing rows"
-	for resourceId in resourceIds.json():
-			resource = (requests.get(repositoryBaseURL + "/resources/" + str(resourceId), headers=headers)).json()
-			if resource["publish"]:
+	
+	#User input to refine functionality of script 
+	print "Welcome to DACSspace!"
```

Comment:
I think it might be best to ask these questions first, that way if the user makes a mistake or decides to cancel out, the script won't do anything. The way this is written now, the script gets the list of IDs and makes the CSV before any user input.